src/main/java/com/bobocode/svydovets/annotation/bean/processor/AutoSvydovetsSetterBeanProcessor.java

```diff
@@ -0,0 +1,68 @@
+package com.bobocode.svydovets.annotation.bean.processor;
+
+import com.bobocode.svydovets.annotation.annotations.AutoSvydovets;
+import com.bobocode.svydovets.annotation.annotations.Qualifier;
+import com.bobocode.svydovets.annotation.bean.factory.BeanFactory;
+import com.bobocode.svydovets.annotation.exception.BeanException;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Map;
+
+/**
+ * Implementation of the {@link BeanProcessor} interface. Executes processing of
+ * {@link AutoSvydovets} marked setters in beans. Such fields will be injected as a
+ * dependency, retrieved from the context.
+ * <p>
+ * If no unique bean is present - processor will try to inject bean based on
+ * {@link Qualifier} value.
+ *
+ * @see BeanProcessor
+ * @see AutoSvydovets
+ * @see Qualifier
+ */
+public class AutoSvydovetsSetterBeanProcessor implements BeanProcessor {
+
+    private final BeanFactory beanFactory;
+
+    public AutoSvydovetsSetterBeanProcessor(BeanFactory beanFactory) {
+        this.beanFactory = beanFactory;
+    }
+
+    @Override
+    public void processBeans(Map<String, Object> rootContext) {
+        for (var entry : rootContext.entrySet()) {
+            Object beanObject = entry.getValue();
+            for (var method : beanObject.getClass().getDeclaredMethods()) {
+                if (isSetter(method)
```

Comment:
Let's use another ordering, since we use AND operator in the implementation we will check the annotation by every setter for the isAnnotationPresent. 
But if we change to
`if (method.isAnnotationPresent(AutoSvydovets.class) && isSetter(method))`
we will want check the isSetter by methods are not marked as @AutoSvydovets